package-lock.json

```diff
@@ -0,0 +1,3410 @@
+{
+  "name": "postcss-value-parser",
+  "version": "3.3.0",
```

Comment:
I would prefer yarn.lock in this repo. npm has bad person as maintainer. I don't trust npm.

Reply:
@TrySound hm, let's do it :smile: Will send next PR